addons/middle-click-popup/addon.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Middle click block dropdown",
+  "description": "Middle Click on the code area, use Ctrl+Space, or use Shift+Space to bring up a floating input box where you can type the name of a block (or parts of it) and drag the block into the code area. Hold Shift while dragging to avoid closing the box when adding multiple blocks at once.",
```

Comment:
`Middle Click` should be `Middle click`

Reply:
Don't repeat `use` twice

Reply:
Maybe the addon name could be more descriptive? I wouldn't be able to deduce what `middle click block dropdown` is.
Maybe we should use some of these words: "search", "find", "search bar", "add block"

Reply:
If in the future the key is rebindable, should 'middle click' really be a part of the addon name at all?

Reply:
@Tacodude7729 other addons use keybinds in their name. we should change them all when we add keybinds. Maybe, not entirely sure how it will work.